src/library/Uno.Themes.WinUI.Markup/Theme/RadioButton.cs

```diff
@@ -0,0 +1,129 @@
+						public static ResourceValue<Brush> Default => new("RadioButtonOuterEllipseFill", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseFillPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("RadioButtonOuterEllipseFillPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseFillPressed")]
+						public static ResourceValue<Brush> Pressed => new("RadioButtonOuterEllipseFillPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseFillDisabled")]
+						public static ResourceValue<Brush> Disabled => new("RadioButtonOuterEllipseFillDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseCheckedFill")]
+						public static ResourceValue<Brush> Checked => new("RadioButtonOuterEllipseCheckedFill", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseCheckedFillPointerOver")]
+						public static ResourceValue<Brush> CheckedPointerOver => new("RadioButtonOuterEllipseCheckedFillPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseCheckedFillPressed")]
+						public static ResourceValue<Brush> CheckedPressed => new("RadioButtonOuterEllipseCheckedFillPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RadioButtonOuterEllipseCheckedFillDisabled")]
+						public static ResourceValue<Brush> CheckedDisabled => new("RadioButtonOuterEllipseCheckedFillDisabled", true);
+					}
+
+					public static class HoverRingFill
+					{
+						[ResourceKeyDefinition(typeof(Brush), "MaterialRadioButtonHoverRingFillPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("MaterialRadioButtonHoverRingFillPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "MaterialRadioButtonHoverRingFillPressed")]
+						public static ResourceValue<Brush> Pressed => new("MaterialRadioButtonHoverRingFillPressed", true);
+					}
+
+					public static class Typography
+					{
+						[ResourceKeyDefinition(typeof(FontFamily), "RadioButtonFontFamily")]
+						public static ResourceValue<FontFamily> FontFamily => new("RadioButtonFontFamily", true);
+
+#if WinUI
+						[ResourceKeyDefinition(typeof(FontWeights), "RadioButtonFontWeight")]
+						public static ResourceValue<FontWeights> FontWeight => new("RadioButtonFontWeight", true);
+#else
+						[ResourceKeyDefinition(typeof(FontWeight), "RadioButtonFontWeight")]
+						public static ResourceValue<FontWeight> FontWeight => new("RadioButtonFontWeight", true);
+#endif
+						[ResourceKeyDefinition(typeof(double), "RadioButtonFontSize")]
+						public static ResourceValue<double> FontSize => new("RadioButtonFontSize", true);
+
+						[ResourceKeyDefinition(typeof(double), "RadioButtonCharacterSpacing")]
+						public static ResourceValue<double> CharacterSpacing => new("RadioButtonCharacterSpacing", true);
+					}
+				}
+			}
+
+			public static class Styles
```

Comment:
Since the styles are moved here as they should, can you remove them from Theme.Syles.cs please  https://github.com/unoplatform/Uno.Themes/blob/868df94042d4922a42e5aaa4f77dd6faef75b32b/src/library/Uno.Themes.WinUI.Markup/Theme.Styles.cs#L139 please